### ISSUE TITLE
abnormal syncytia formation

### ISSUE THREAD

**Issue by yongqunh** [obophenotype/human-phenotype-ontology#7473]
**Preferred term label:**
abnormal syncytia formation

https://en.wikipedia.org/wiki/Syncytium

**Synonyms**
abnormal syncytium
abnormal symplasm  

**Definition (free text, please give PubMed ID)**
An abnormal formation of syncytium or symplasm, a multinucleate resulting from multiple cell fusions of uninuclear cells (i.e., cells with a single nucleus).

note: this is found in COVID-19: 
https://www.embopress.org/doi/full/10.15252/embj.2020106267 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

SARS-CoV-2
https://www.embopress.org/doi/full/10.15252/embj.2020106267
(note:  this is the term we would like to use in CIDO but it does not exist in HPO yet)

**Your nano-attribution (ORCID)**
http://orcid.org/0000-0001-9189-9661 (Oliver He)

Anthony Huffman and Oliver He

**Comment by pnrobinson** [comment]
I think this is a little outside the scope of HPO, which is for clinical phenotypes. I think that this is an experimental observation that is not used at all in clinical contexts. I am not sure where the correct place is for this -- possibly it should go directly into Coronavirus Infectious Disease Ontology?

**Comment by yongqunh** [comment]
I see. thanks.
